test/services/ReviewServiceSpec.scala

```diff
@@ -30,7 +33,9 @@ class ReviewServiceSpec extends BaseSpec {
 
   private trait Test extends MockReviewConnector with ReviewData {
 
-    implicit val hc = HeaderCarrier()
+    implicit val hc: HeaderCarrier = HeaderCarrier()
+    val approvalProcessSummary: ApprovalProcessSummary = ApprovalProcessSummary("id", "title", LocalDate.now, ApprovalStatus.Published)
```

Comment:
I need to do this to, but reading Programming in Scala 4th Edition, the style in Scala is to omit the type definition if it is obvious. When using a simple apply I think we can omit the type in future.